### DISCUSSION TITLE
【求助】有没有战斗爽的好心绳匠帮忙打下我们学校，急！！！

### DISCUSSION THREAD

**Discussion by ywbsiowjn** [share121/inter-knot#2190] [其他]
![Screenshot_20240815_100945_com miHoYo Nap](https://github.com/user-attachments/assets/94fb5986-dbfd-4f34-a352-bb4797df68e5)
![Screenshot_20240815_135219_com tencent mm](https://github.com/user-attachments/assets/6ac534aa-67be-46db-a814-bc88fda70a47)
本人于绳网发现此委托，评论区尽数战斗爽，朋友便心生一计，因个人能力孱弱，遂发绳网寻求帮助，感谢！

**Comment by SuiHanjun** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >
这个委托我拿…………不下

**Comment by poolofwater** [comment]
找不到这个地方欸，这是哪个未发现的地方吗<img src="https://img-static.mihoyo.com/communityweb/upload/59fe234835e808c41b0208c50be44914.png" alt="偶像姬-大声bb" width="75" >

**Comment by ElvenShark** [comment]
这个委托我接下了

**Comment by Kamisato100** [comment]
这个委托我不接<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
